dbt_project.yml

```diff
@@ -191,13 +191,18 @@ models:
       ethereum:
         +schema: curvefi_ethereum
         +materialized: view
-
     ovm1:
       +schema: ovm1
       +materialized: table
       optimism:
         +schema: omv1_optimism
         +materialized: table
+    cow_protocol:
+      +schema: cow_protocol
+      +materialized: view
+      ethereum:
+        +schema: cow_protocol_ethereum
+        +materialized: view
```

Comment:
I am not sure how all the labels work just yet, but I was expecting the solvers to be the materialized view... this makes it look like the entire namespace is materialized view... I see that ovm1 uses `+materialized: table` just above and above that is also `+materialized: view`...

Reply:
In this case, the +materialized: view means all models in these repositories will be views by default (which is usually a good idea). However, it's very easy to materialize specific models as tables/incremental tables within each model in the config JINJA statement

Reply:
> In this case, the +materialized: view means all models in these repositories will be views by default (which is usually a good idea). However, it's very easy to materialize specific models as tables/incremental tables within each model in the config JINJA statement

Does the one in the model config overwrite the general one?

Reply:
Yes, ideally you can remove the `materialized='view'` inside the jinja config in the model file so you avoid defining materialization twice (although it won't break things if you keep things as such)